spec/takeaway_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'takeaway'
+
+describe Takeaway do
+  it "returns a menu when requested" do
+    expect(subject).to respond_to(:read_menu)
+  end
+
+  it "returns the full menu when requested" do
+    expect(subject.menu.dishes.length).to eq 5
+  end
+
+  describe "#order" do
+    it "responds to order" do
+      expect(subject).to respond_to(:place_order).with(2).arguments
+    end
+
+    it "adds an order to the basket" do
+      subject.place_order("pizza")
+      expect(subject.order.length).to eq 1
+    end
```

Comment:
Can we also assert that `subject.place_order` returns the correct output? i.e. `"1 x pizza(s) added to your order"`